src/servers/middlewares/auth-router.ts

```diff
@@ -1,12 +1,14 @@
-import express from "express"
+import { Configuration, V0alpha2Api, V0alpha2ApiInterface } from "@ory/client"
 import cors from "cors"
-import { Configuration, V0alpha2ApiInterface, V0alpha2Api } from "@ory/client"
+import express from "express"
 
 import { Users } from "@app"
-import { baseLogger } from "@services/logger"
-import { isDev, getKratosConfig } from "@config"
+import { getKratosConfig, isDev, JWT_SECRET } from "@config"
 import { parseIps } from "@domain/users-ips"
 import { mapError } from "@graphql/error-map"
+import { baseLogger } from "@services/logger"
+
+import * as jwt from "jsonwebtoken"
```

Comment:
please move to the beginning of the file

Reply:
any clue why this is not enforced by our linter?